### ISSUE TITLE
x/build/cmd/gopherbot: reply to commit comments saying we do not monitor them

### ISSUE THREAD

**Issue by FiloSottile** [golang/go#28296]
We get from time to time a comment on a commit in the main repo. We do not monitor those and nobody gets a ping for them (except the author of the commit who might or might not check GitHub notifications and does not have to). gopherbot should reply automatically pointing them towards the issue tracker and https://github.com/golang/go/wiki/Questions.

(Almost sure there was an issue for this, but couldn't find it. Please close as dup if anyone does.)

**Comment by adamdecaf** [comment]
The CL mentioned lowering our freeze timeout for closed issues instead. Right now the timeout is [365 days](https://github.com/golang/build/blob/b3b29ee65eb7b95036c0a3c9921b92ee427ca65c/cmd/gopherbot/gopherbot.go#L686). 30 days? 90?

**Comment by ianlancetaylor** [comment]
I say 30 days, and make sure that the freeze notice mentions https://golang.org/wiki/Questions .

**Comment by ianlancetaylor** [comment]
Good point.  I was thinking that there was a notice on the issue when we freeze it, but of course you're right that there isn't.  In that case, sure, 90 days seems plausible to me.

Sorry for the runaround.

**Comment by adamdecaf** [comment]
What about adding the freeze notice only to issues that have had a comment after the issue is closed?

**Comment by andybons** [comment]
So 90 days without a comment on the issue?

**Comment by dmitshur** [comment]
Merging this into proposal #25450.
